Rakefile

```diff
@@ -0,0 +1,11 @@
+task :ci => [:dump, :test]
```

Comment:
This whole thing was pretty copy-pasted from http://whileimautomaton.net/2013/02/08211255.

Since I'm using `vim-flavor`, which is a gem, to manage those dependencies, there's already a Ruby dependency, so I didn't see any problem with also using Rake.